src/droid-vibrator.c

```diff
@@ -52,6 +234,9 @@ droid_vibrator_sink_shutdown (NSinkInterface *iface)
 {
     (void) iface;
     N_DEBUG (LOG_CAT "sink shutdown");
+
+    g_hash_table_destroy (plugin_effects);
```

Comment:
Just wondering whether if this can get called while plugin_effects is null and whether it is ok to call g_hash_table_destroy() with null arg?